frontend/src/components/ProjectList/components/Dataset/components/MoreInformation/index.tsx

```diff
@@ -1,31 +1,47 @@
-import { FC } from "react";
+import React, { FC } from "react";
 import { Project } from "src/common/entities";
+import { StyledAnchor, Wrapper } from "./style";
+
+interface AnchorProps {
+  url: string;
+}
+
+const Anchor: FC<AnchorProps> = ({ url, children }) => {
+  return (
+    <StyledAnchor href={url} target="_blank" rel="noopener">
+      {children}
+    </StyledAnchor>
+  );
+};
 
 interface Props {
   links: Project["links"];
 }
 
-const MoreInformation: FC<Props> = () => {
-  return null;
-
-  // (thuang): Temporarily comment this out until BE has `link.name`
-  // return (
-  //   <Wrapper>
-  //     {links.map(link => {
-  //       return (
-  //         <StyledAnchor
-  //           key={link.url}
-  //           href={link.url}
-  //           target="_blank"
-  //           rel="noopener"
-  //         >
-  //           {link.name}
-  //           <br />
-  //         </StyledAnchor>
-  //       );
-  //     })}
-  //   </Wrapper>
-  // );
+const MoreInformation: FC<Props> = ({ links }) => {
+  const uniqueLinks = Array.from(
```

Comment:
Seems like there are duplicate links, so this dedupes them

<img width="814" alt="Screen Shot 2020-09-09 at 10 22 29 AM" src="https://user-images.githubusercontent.com/6309723/92635508-ce5f9280-f28a-11ea-9b31-c117d072a9e6.png">